example/scripts/tests-parallel.sh

```diff
@@ -14,12 +15,13 @@ testOutputDir=scripts/test_output
 rm -r -f $testOutputDir
 mkdir -p $testOutputDir
 
-./node_modules/.bin/parallel sh scripts/run_test.sh  $testOutputDir ::: $TESTS
+./node_modules/.bin/parallel -j 4 sh scripts/run_test.sh  $testOutputDir ::: $TESTS
```

Comment:
Instead of hardcoding the number "4", would it be better to base it off of the number of CPUs?
```suggestion
./node_modules/.bin/parallel -j $(node -e 'console.log(os.cpus().length + 1)') sh scripts/run_test.sh  $testOutputDir ::: $TESTS
```

Reply:
The default is number of CPUs